libs/utils/include/utils/NameComponentManager.h

```diff
@@ -49,29 +49,78 @@ class SafeString {
 } // namespace details
 
 
+/**
+ * \class NameComponentManager NameComponentManager.h utils/NameComponentManager.h
+ * \brief Allows clients to associate string labels with entities.
+ *
+ * To access the name of an existing entity, clients should first use NameComponentManager to get a
+ * temporary handle called an \em instance. Please note that instances are ephemeral; clients should
+ * store entities, not instances.
+ *
+ * Usage example:
+ *
+ *      auto names = new NameComponentManager(EntityManager::get());
```

Comment:
Surround with ~~~~ ?

Reply:
It already looks pretty good, it detects the indentation and used a code font.

Reply:
Ah, using tildes instead of indentation enables syntax highlighting.  Thanks!